velox/expression/VectorUdfTypeSystem.h

```diff
@@ -72,6 +72,11 @@ struct resolver<Row<T...>> {
   using out_type = core::RowWriter<typename resolver<T>::out_type...>;
 };
 
+template <typename... T>
+struct resolver<RowWriterT<T...>> {
+  using out_type = exec::RowWriter<T...>;
```

Comment:
nit: I don't think you need the namespace here, just
  using out_type = RowWriter<T...>;

Reply:
there is a core::RowWriter, but yeh i can remove that.

Reply:
```
template <typename... T>
struct resolver<Row<T...>> {
  using in_type = RowView<true, T...>;
  using null_free_in_type = RowView<false, T...>;
  using out_type = core::RowWriter<typename resolver<T>::out_type...>;
};

template <typename... T>
struct resolver<RowWriterT<T...>> {
  using out_type = exec::RowWriter<T...>;
};
```
just to distinguish from core::RowWriter.

but i can remove it.